config.py

```diff
@@ -309,6 +311,10 @@ def show_staff_picks_on_top_level(cls):
         return cls.policy(cls.SHOW_STAFF_PICKS_ON_TOP_LEVEL, default=True)
 
     @classmethod
+    def localization_languages(cls):
+        return cls.policy(cls.LOCALIZATION_LANGUAGES, default=["en"])
```

Comment:
We generally use the 3-letter language codes in the config file but I see why you're using the 2-letter codes here--that's what Flask-Babel's request.accept_languages_best_match takes. Is it worth taking 3-letter codes for the sake of consistency and using LanguageCodes.three_to_two to convert them?